src/js/components/stock-movement-wizard/outbound/AddItemsPage.jsx

```diff
@@ -441,7 +442,7 @@ class AddItemsPage extends Component {
     const lineItemsToBeUpdated = [];
     _.forEach(lineItemsWithStatus, (item) => {
       // We wouldn't update items with quantity requested < 0
-      if (!item.quantityRequested || parseInt(item.quantityRequested, 10) < 0) {
+      if (!item.quantityRequested === undefined || parseInt(item.quantityRequested, 10) < 0) {
```

Comment:
I am not convinced about that - is `undefined` the only "empty" value `quantityRequested` can be? Can't it be `''` or `null`? 
I know it might turn into "spaghetti", but wouldn't it be more safe to do `(item.quantityRequested != 0 && !item.quantityRequested)`

If there is no way `quantityRequested` can be an empty string/null, then I'm fine with that

Reply:
Oh, good catch! Thanks